### PR TITLE
chore: auto assign installed issues  for verification

### PR DESCRIPTION
**Pull request by benelan**
## Summary
- setting the `3 - installed` label on an issue automatically removes all currently assigned users and assigns me

### CODE REVIEW

**Comment by benelan** [comment]
tested here: https://github.com/benelan/test-actions/issues/38

File: .github/workflows/assign-for-verification.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                body: "This issue is installed, assigning to @benelan for verification",
+              });
+              await github.issues.update({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                assignees: ['benelan']
```
Comment (jcfranco) [inline .github/workflows/assign-for-verification.yml:24]: Nitpick: can you use double quotes for consistency? Sidebar, looks like we also need to include `yml` files in our linting.

**Review by jcfranco — APPROVED** [review]
🦾 🤖  #benditlikebender

File: .github/workflows/assign-for-verification.yml (hunk 1 above)
Comment (benelan) [reply .github/workflows/assign-for-verification.yml]: I did a find/replace for all of the yml workflow files, I'm a nitpicker too. I'll look into the linting.

**Pull request merged**